app/main.py

```diff
@@ -341,6 +348,17 @@ def _dump_cert(self, secret_value, cert_filename, key_filename, content_type):
         with open(cert_path, 'w') as cert_file:
             for cert in certs:
                 cert_file.write(crypto.dump_certificate(crypto.FILETYPE_PEM, cert))
+        
+        if ca_certs != (): 
```

Comment:
Just write "if ca_certs:" - slightly better python style.  (An empty list evaluates to false)